### PR TITLE
15.0 mig stock valuation no developer mode

### PR DESCRIPTION
**Pull request by olveracuenca**

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]

File: stock_valuation_no_developer_mode/views/stock_valuation_layer.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    <record id="stock_valuation_layer_picking_view" model="ir.ui.view">
+        <field name="name">stock.valuation.layer.picking.view</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock_account.stock_valuation_layer_picking" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//button[@name='action_view_stock_valuation_layers']"
+                position="attributes"
+            >
+                <attribute name="groups">account.group_account_invoice</attribute>
```
Comment (flotho) [inline stock_valuation_no_developer_mode/views/stock_valuation_layer.xml]: won't it be useful to also add inventory manager group ?

Comment (rousseldenis) [reply stock_valuation_no_developer_mode/views/stock_valuation_layer.xml]: I agree. Stock valuation layer model access is only defined for inventory manager.

If you add just this and people in billing group are not in the inventory manager one, clicking on button will raise an access error

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by rousseldenis** [comment]
You should also respect commit history.

See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0

**Comment by rousseldenis** [comment]
@olveracuenca Could you fix this and respect commits history ?

**Comment by rousseldenis** [comment]
@olveracuenca Could you attend comments especially about commit history ?

Thanks

**Comment by olveracuenca** [comment]
@rousseldenis ok, ill work on it right now and update it.
Thanks.

**Comment by cvinh** [comment]
@olveracuenca do you need some help ?

**Pull request closed without merging**